presto-main/src/main/java/com/facebook/presto/server/testing/TestingPrestoServer.java

```diff
@@ -339,7 +339,7 @@ public ConnectorId createCatalog(String catalogName, String connectorName)
     public ConnectorId createCatalog(String catalogName, String connectorName, Map<String, String> properties)
     {
         ConnectorId connectorId = connectorManager.createConnection(catalogName, connectorName, properties);
-        updateConnectorIdAnnouncement(announcer, connectorId);
```

Comment:
Typo: `Fix race condition in TestQueryStateInfoResource` and that might make a better title than message.